service/accounts/openstack_manager.py

```diff
@@ -24,17 +30,31 @@
 from core.ldap import get_uid_number
 from core.models.identity import Identity
 
-from service.accounts.base import CachedAccountDriver
+from service.accounts.base import BaseAccountDriver
+
+
+def get_unique_id(userid):
+    if 'iplantauth.authBackends.LDAPLoginBackend' in settings.AUTHENTICATION_BACKENDS:
+        return get_uid_number(userid)
+    else:
+        return get_random_uid(userid)
+
+def get_random_uid(userid):
+    """
+    Given a string (Username) return a value < MAX_SUBNET
+    """
+    MAX_SUBNET = 4064
+    return int(random.uniform(1, MAX_SUBNET))
 
 
-class AccountDriver(CachedAccountDriver):
+class AccountDriver(BaseAccountDriver):
     user_manager = None
     image_manager = None
     network_manager = None
     core_provider = None
 
     MASTER_RULES_LIST = [
-        ("ICMP", 0, 255),
+        ("ICMP", -1, -1),
```

Comment:
I have no idea what `-1` means here. I assume that someone would know that this is the Internet Control Message Protocol value?